<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>337</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>211</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>QListWidget</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>当前项变化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QCheckBox" name="m_checkBox_ListEditable">
             <property name="text">
              <string>可编辑</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="m_List_toolButton_Selection">
             <property name="text">
              <string>...</string>
             </property>
             <property name="popupMode">
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_List_toolButton_SelectAll">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_List_toolButton_CheckOutAll">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="m_List_toolButton_InvertSelection">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>New Item</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
          <item>
           <property name="text">
            <string>New Item2</string>
           </property>
           <property name="checkState">
            <enum>Checked</enum>
           </property>
           <property name="flags">
            <set>ItemIsSelectable|ItemIsEditable|ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>QTreeWidget</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>QTableWidget</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="minimumSize">
       <size>
        <width>146</width>
        <height>245</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>146</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QWidget" name="toolBoxPage1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>146</width>
         <height>233</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <attribute name="label">
        <string>QListWidget操作</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolButton" name="m_toolButton_Reset">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_toolButton_Clear">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_toolButton_Insert">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_toolButton_Add">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_toolButton_Delete">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="toolBoxPage2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>146</width>
         <height>233</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QTreeWidget操作</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>146</width>
         <height>233</height>
        </rect>
       </property>
       <attribute name="label">
        <string>QTableWidget操作</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>609</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResetList"/>
   <addaction name="actionClearList"/>
   <addaction name="actionInsertEntry"/>
   <addaction name="actionAddItem"/>
   <addaction name="actionDeleteItem"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionResetList">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Reset.ico</normaloff>:/ico/Reset.ico</iconset>
   </property>
   <property name="text">
    <string>ResetList</string>
   </property>
  </action>
  <action name="actionClearList">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Clear.ico</normaloff>:/ico/Clear.ico</iconset>
   </property>
   <property name="text">
    <string>ClearList</string>
   </property>
  </action>
  <action name="actionInsertEntry">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Insert.ico</normaloff>:/ico/Insert.ico</iconset>
   </property>
   <property name="text">
    <string>InsertEntry</string>
   </property>
  </action>
  <action name="actionAddItem">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Add.ico</normaloff>:/ico/Add.ico</iconset>
   </property>
   <property name="text">
    <string>AddItem</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Delete.ico</normaloff>:/ico/Delete.ico</iconset>
   </property>
   <property name="text">
    <string>DeleteItem</string>
   </property>
  </action>
  <action name="actionItemSelection">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/Selection.ico</normaloff>:/ico/Selection.ico</iconset>
   </property>
   <property name="text">
    <string>ItemSelection</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="ico.qrc">
     <normaloff>:/ico/exit.ico</normaloff>:/ico/exit.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>SelectAll</string>
   </property>
  </action>
  <action name="actionCheckOutAll">
   <property name="text">
    <string>CheckOutAll</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="text">
    <string>InvertSelection</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionItemSelection</sender>
   <signal>triggered()</signal>
   <receiver>actionInvertSelection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
